### PR TITLE
doc: west: add instructions for moving to west and troubleshooting

### PR DESCRIPTION
**Pull request by mbolivar**
These are based on user feedback on slack and the mailing list.

### CODE REVIEW

**Comment by yashi** [comment]
Love it!  Thanks, @mbolivar !

File: doc/guides/west/moving-to-west.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+   The name :file:`zephyrproject` is arbitrary. You can choose any name,
+   as long as there are no spaces anywhere in the path.
+
+#. Create a :ref:`west installation <west-installation>` using the zephyr
+   repository as a local manifest repository::
+
+     west init -l zephyr
+     west update
+
+Troubleshooting
```
Comment (carlescufi) [inline doc/guides/west/moving-to-west.rst]: I don't think this section belongs here.

I have 2 suggestions:

1. Let's create a global FAQ. I think that is the better approach
2. Let's create a west Troubleshooting section that is a separate page

**Review by carlescufi — CHANGES_REQUESTED** [review]

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Comment by mbolivar** [comment]
@carlescufi @dbkinder comments addressed; thanks!

**Review by dbkinder — APPROVED** [review]
Works for me. Thanks.

**Review by carlescufi — APPROVED** [review]

**Pull request merged**